cmd/autoowners/main.go

```diff
@@ -142,14 +142,14 @@ func getOwnersHTTP(orgRepo orgRepo) (httpResult, error) {
 	for _, filename := range []string{"OWNERS", "OWNERS_ALIASES"} {
 		data, err := gc.GetFile(orgRepo.Organization, orgRepo.Repository, filename, "")
 		if err != nil {
-			if _, nf := err.(*github.FileNotFound); nf {
-				logrus.WithField("orgRepo", orgRepo.repoString()).WithField("filename", filename).
-					Debug("Not found file in the upstream repo")
-				httpResult.ownersFileExists = false
-				continue
-			} else {
-				return httpResult, err
-			}
+			return httpResult, err
+		}
+
+		if _, ok := err.(*github.FileNotFound); ok {
```

Comment:
This would change the logic ...
When the error is with type `FileNotFound` ... we do warning logs only, instead of returning error.
Because we want to keep the caller of `getOwnersHTTP` going for other repos.

During the review, I do think we have a bug in this function.
I will add tests with another PR for this function and ask for your review.

Reply:
https://github.com/openshift/ci-tools/pull/176